### PR TITLE
DOC Document adding TopPage extensions by default

### PR DESCRIPTION
**Pull request by GuySartorelli**
Documents changes in https://github.com/silverstripe/silverstripe-elemental/pull/1010

## Parent issue
- https://github.com/silverstripe/silverstripe-elemental/issues/1009

### CODE REVIEW

File: en/04_Changelogs/5.0.0.md [hunk 1]
```diff
@@ -136,6 +136,7 @@ This empowers advanced `Extension` functionality such as [Versioned::canPublish(
 ### Other new features {#other-features}
 
 - [DataObject::get_one()][api:SilverStripe\ORM\DataObject::get_one()] can now be called directly from subclasses of `DataObject` without passing in a class as the first argument (e.g. `SiteTree::get_one(filter: ['Title:startsWith' => 'About'])`).
+- The elemental [TopPage DataExtension](api:DNADesign\Elemental\TopPage\DataExtension) is applied to [BaseElement](api:DNADesign\Elemental\Models\BaseElement) and [ElementalArea](api:DNADesign\Elemental\Models\ElementalArea) by default, and the corresponding [SiteTreeExtension](api:DNADesign\Elemental\TopPage\SiteTreeExtension) is applied to [SiteTree](api:SilverStripe\CMS\Model\SiteTree) by default. This allows you to use `$TopPage` in your elemental block templates as a more performant alternative to `$Page` when referring to the page the element belongs to.
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/5.0.0.md:224]: This is incorrect for the way this has been implemented - those extensions are _not_ being applied, they've been completely replaced.

Comment (emteknetnz) [reply en/04_Changelogs/5.0.0.md:224]: We're now doing this via extensions again, so I'd say the docs here are correct

**Review by emteknetnz — APPROVED** [review]

**Comment by emteknetnz** [comment]
@sabina-talipova you'll need to rebase to resolve merge conflicts

File: en/04_Changelogs/4.13.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: 4.13.0 (unreleased)
```
Comment (ChloeSartorelli) [inline en/04_Changelogs/4.13.0.md]: This changelog will need to be added in a separate PR targeting the `4` branch

Comment (sabina-talipova) [reply en/04_Changelogs/4.13.0.md]: DONE

**Review by emteknetnz — APPROVED** [review]
